flink-python/src/main/java/org/apache/flink/table/runtime/operators/python/AbstractPythonScalarFunctionOperator.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.runtime.operators.python;
```

Comment:
The current package naming conversion follows the strategy defined in https://issues.apache.org/jira/browse/FLINK-12803. For example, the package name of operators defined in runtime is "org.apache.flink.streaming.api.operators". Then for operators defined in Python, it will be "org.apache.flink.streaming.api.operators.python".

Reply:
Thanks for the point. Maybe we can put all runner related classes under runtime/runner/python? The nature of these classes is a runner which runs a function instead of a function by itself. What do you think?

Reply:
Make sense. +1